### DISCUSSION TITLE
Room 10 - Series 3

### DISCUSSION THREAD

**Discussion by abaumane** [IBAHCM/RPiR#206] [Archive]
Agate Baumane - @abaumane 
Valeria Patricia Torres Espin - @valeria-torres-espin
Xinyue Jia - @XinyueJia

**Comment by abaumane** [comment]
My series 3 repo - https://github.com/IBAHCM/abaumaneSeries03

**Comment by abaumane** [comment]
@valeria-torres-espin @XinyueJia hey girls I noticed you hadn't put your repo links in here!

**Reply by XinyueJia** [reply]
Mine - https://github.com/IBAHCM/XinyueJiaSeries03
